projects/addon-doc/components/navigation/navigation.component.ts

```diff
@@ -55,9 +53,26 @@ import {
     providers: NAVIGATION_PROVIDERS,
 })
 export class TuiDocNavigationComponent {
+    // private readonly cdr = inject(ChangeDetectorRef);
+    // private readonly titleService = inject(Title);
+    private readonly router = inject(Router);
+    // private readonly destroy$ = inject(TuiDestroyService, {self: true});
+    // private readonly title$ = inject(NAVIGATION_TITLE);
+    // private readonly readyToScroll$ = inject(TUI_DOC_PAGE_LOADED);
```

Comment:
Could you explain these comments, please ?

Reply:
fixed